doc/ipython-notebooks/neuralnets/autoencoders.ipynb

```diff
@@ -1,471 +1,484 @@
 {
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deep Autoencoders"
+   ]
+  },
```

Comment:
something is very weird here. Why all these changes in the diff?
I expected single line changes...ideas?